### ISSUE TITLE
Installation of additional sources does not work

### ISSUE THREAD

**Issue by simonbcn** [filiparag/wikiman#30]
```
Arch Linux
wikiman v2.12.1 (installed from AUR)
```

The arch-wiki source is installed with `pacman -S arch-wiki-docs`
Installation of additional sources does not work:
```
~/fuentes $ make -f ./wikiman-makefile source-gentoo
% Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
Dload  Upload   Total   Spent    Left  Speed
100   655  100   655    0     0   1523      0 --:--:-- --:--:-- --:--:--  1523
100  122M  100  122M    0     0  13.4M      0  0:00:09  0:00:09 --:--:-- 15.2M

~/fuentes $ sudo make -f ./wikiman-makefile source-install
cp: can't be done `stat' on '/home/juan/fuentes/srcbuild/usr/local/share/doc': File or directory does not exist
make: *** [wikiman-makefile:132: source-install] Error 1

~/fuentes $ sudo make -f ./wikiman-makefile clean

~/fuentes $ wikiman -S
NAME       STATE  PAGES  PATH
arch         +     6363  /usr/share/doc/arch-wiki/html
fbsd                     (not installed)
gentoo                   (not installed)
man          +    16862  /usr/local/man /usr/local/share/man /usr/share/man /usr/lib/jvm/default/man
tldr                     (not installed)

$ ls /usr/share/wikiman/sources
arch.sh  fbsd.sh  gentoo.sh  man.sh  tldr.sh
```

The same when I try to install `source-tldr`.

On the other hand, I think it would be better to be able to do this with an only `wikiman` option.

**Comment by filiparag** [comment]
Bug should be fixed in [fa286f3](https://github.com/filiparag/wikiman/commit/fa286f369b8afa111cf641c17252233e2ee4bbf1). Please report if it persists.
